### PR TITLE
Added notebook for livedata via Mantid

### PR DESCRIPTION
**Pull request by matthew-d-jones**
Adds a notebook demonstrating consuming data from Kafka via Mantid and documenting the limitation that plots in the notebook cannot be updated during consumption of data.

Closes https://github.com/scipp/scipp/issues/932

### CODE REVIEW

**Comment by matthew-d-jones** [comment]
Wait until #10 is merged and then add new notebook to the exclude list in `.azure-pipelines/master.yml` as streaming notebook cannot be tested in CI.

**Review by SimonHeybrock — APPROVED** [review]

**Pull request merged**
